### PR TITLE
Fix/asari buildings

### PR DESCRIPTION
**Pull request by maxim-a-k**
Updated all buildings that the Asari can build to produce 0.2 Unity and 1 Amenity based on their unique starting tech  Asari Architecture.

### CODE REVIEW

**Comment by maxim-a-k** [comment]
Just testing it now something seems to have caused a big issue with the UI loading? tbd

File: common/buildings/00_capital_buildings.txt [hunk 1]
```diff
@@ -247,6 +275,12 @@ building_capital = {
 
 	resources = {
 		category = planet_buildings
+		produces = {
+			trigger = { exists = owner
```
Comment (cemal-efe-g) [inline common/buildings/00_capital_buildings.txt:279]: I might be wanting too much, but can you add a linebreak after trigger = {

It is making it really hard to read.

File: localisation/english/btr_buildings_l_english.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
  ### Asari Republic Buildings ###
  
- btr_building_asrep_comm_center:0 "Republican Communication Center"
+ btr_building_asrep_comm_center:0 "Communication Centre of the Republics"
```
Comment (cemal-efe-g) [inline localisation/english/btr_buildings_l_english.yml:5]: @puckhead73 Can you confirm this change? Its English just seems weird to me.

**Review by cemal-efe-g — CHANGES_REQUESTED** [review]
I have requested a couple of things, nothing much hopefully.

File: localisation/english/btr_buildings_l_english.yml (hunk 2 above)
Comment (maxim-a-k) [reply localisation/english/btr_buildings_l_english.yml:5]: @puckhead73 feel free if you think it should be different this is just what Mis provided after we pointed out the Republican thing.

**Comment by mstonefield** [comment]
Irrelevant due to already being fixed in AI overhaul

**Comment by maxim-a-k** [comment]
I agree with Tex we should come back to this later

**Pull request closed without merging**